documentation/staging/content/samples/azure-kubernetes-service/includes/create-aks-cluster-storage.txt

```diff
@@ -1,86 +1,147 @@
-# Create file share
-$ az storage share create -n $AKS_PERS_SHARE_NAME --connection-string $AZURE_STORAGE_CONNECTION_STRING
-```
+    $ az storage account create \
+        --resource-group $AKS_PERS_RESOURCE_GROUP \
+        --name $AKS_PERS_STORAGE_ACCOUNT_NAME \
+        --location $AKS_PERS_LOCATION \
+        --sku Premium_LRS \
+        --kind FileStorage \
+        --https-only false \
+        --default-action Deny
+    ```
 
-Successful output will be exactly the following:
+    Successful output will be a JSON object with the entry `"type": "Microsoft.Storage/storageAccounts"`.
 
-```shell
-{
-  "created": true
-}
-```
+2. Create a NFS share
 
-The operator uses Kubernetes Secrets.  We need a storage key for the secret. These commands query the storage account to obtain the key, and then stores the storage account key as a Kubernetes secret.
+    To create the file share, you must use `NoRootSquash` to allow the operator to change the ownership of the directory in NFS share.
 
-```shell
-$ export STORAGE_KEY=$(az storage account keys list --resource-group $AKS_PERS_RESOURCE_GROUP --account-name $AKS_PERS_STORAGE_ACCOUNT_NAME --query "[0].value" -o tsv)
-```
+    Otherwise, you will get an error like `chown: changing ownership of '/shared': Operation not permitted`.
 
-Verify the successful output by examining the `STORAGE_KEY` environment variable.  It must not be empty.  It must be a long ASCII string.
+    The following command creates a NFS share with 100GiB:
 
-We will use the `kubernetes/samples/scripts/create-kubernetes-secrets/create-azure-storage-credentials-secret.sh` script to create the storage account key as a Kubernetes secret, naming the secret with value `${NAME_PREFIX}azure-secret`. Please run:
+    ```shell
+    # Change value as needed for your own environment
+    $ export AKS_PERS_SHARE_NAME="${NAME_PREFIX}-weblogic-${TIMESTAMP}"
+    # Create NFS file share
+    $ az storage share-rm create \
+        --resource-group $AKS_PERS_RESOURCE_GROUP \
+        --storage-account $AKS_PERS_STORAGE_ACCOUNT_NAME \
+        --name ${AKS_PERS_SHARE_NAME} \
+        --enabled-protocol NFS \
+        --root-squash NoRootSquash \
+        --quota 100
+    ```
 
-```shell
-# Please change persistentVolumeClaimNameSuffix if you changed pre-defined value "regcred" before generating the configuration files.
-$ export SECRET_NAME_AZURE_FILE="${NAME_PREFIX}azure-secret"
+    The command provisions a NFS file share with NFS 4.1 or above.
 
-#cd kubernetes/samples/scripts/create-kubernetes-secrets
-$ ./create-azure-storage-credentials-secret.sh -s $SECRET_NAME_AZURE_FILE -a $AKS_PERS_STORAGE_ACCOUNT_NAME -k $STORAGE_KEY
-```
+3. Configure network security
 
-You will see the following output:
+    We have to configure the network security allowing access from AKS cluster to the storage account.
```

Comment:
Please make the same edits here that I suggested previously for these statements.

Reply:
See https://github.com/galiacheng/weblogic-kubernetes-operator/commit/2b18f06d51752b28a22ec323464107e1d3305788